doc/ref/modules/all/index.rst

```diff
@@ -60,7 +60,7 @@ execution modules
     boto_datapipeline
     boto_dynamodb
     boto_ec2
-    boto_efs
+    boto3_efs
```

Comment:
please add boto_efs back to this, also dropping boto3_efs with the other boto3 modules would be good.

Reply:
reverted and done!